src/System.Private.Xml/tests/XmlReader/XmlResolver/XmlSystemPathResolverTests.cs

```diff
@@ -13,9 +13,11 @@ public class XmlSystemPathResolverTests
         private const int k_getUniqueFileNameAttempts = 10;
 
         [Fact]
-        [SkipOnTargetFramework(TargetFrameworkMonikers.Uap)]  //[ActiveIssue(13121)]  // Access to path is denied in UWP
         public static void TestResolveRelativePaths()
         {
+            if (PlatformDetection.IsWinRT)  // Access to path not authorized inside an AppContainer
```

Comment:
Ah, that's because it's relative to the current directory which is not accessible.
If we want to test resolving relative paths, I suggest changing current directory (just for this test) to point to GetTempPath(). Then the test should work (without your change here)